### PR TITLE
DaxiangLi: add checking to foo.py

### PR DESCRIPTION
**Pull request by DaxiangLi**
HuanianLi: I added checking to foo.py, please review, thanks.

### CODE REVIEW

File: foo.py [hunk 1]
```diff
@@ -3,6 +3,10 @@
 import sys
 
 def main(argc, argv):
+    if argc != 2:
+        sys.stderr.write("Usage: %s <num>\n" % argv[0])
+        return 1
```
Comment (HuanianLi) [inline foo.py]: Please use 'return -1' instead at L8.

**Review by HuanianLi — CHANGES_REQUESTED** [review]

File: foo.py (hunk 1 above)
Comment (DaxiangLi) [reply foo.py]: Done. Please check again.

**Review by HuanianLi — CHANGES_REQUESTED** [review]
Hi Daxiang! If you get to the final version of the pull, can you please squash the commits into a single one and let me know so I can check it?

**Comment by DaxiangLi** [comment]
The two commits are squashed into a single one now, please check again.

**Comment by DaxiangLi** [comment]
HuanianLi, I have resolved the conflict in foo.py, please review and merge the code, thanks.

**Pull request merged**